keyboards/1upkeyboards/1upslider8/info.json

```
@@ -0,0 +1,77 @@
+{
+    "manufacturer": "1upkeyboards",
+    "keyboard_name": "1upslider8",
+    "maintainer": "ziptyze",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "board": "GENERIC_RP_RP2040",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x5611",
+        "vid": "0x6F75"
+    },
+    "dynamic_keymap": {
+        "layer_count": 10
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false,
+        "oled": true,
+        "midi":true,
+        "rgb_matrix": true
+    },
+    "encoder": {
+        "enabled": true,
+        "rotary": [
+            {
+                "pin_a": "GP21",
+                "pin_b": "GP22"
+            }
+        ]
+    },
+    "matrix_pins": {
+        "direct": [
+            ["GP27", "GP17", "GP18", "GP19", "GP20", "GP15", "GP14", "GP13", "GP16"]
+        ]
+    },
+    "bootmagic": {
+        "enabled": true,
+        "matrix": [0, 1]
+    },
+    "ws2812": {
+        "pin": "GP26"
+    },
+    "rgb_matrix": {
+        "driver": "WS2812",
+        "layout": [
+        { "flags": 4, "matrix": [0, 1], "x": 28, "y": 16 },
+        { "flags": 4, "matrix": [0, 2], "x": 84, "y": 16 },
+        { "flags": 4, "matrix": [0, 3], "x": 140, "y": 16 },
+        { "flags": 4, "matrix": [0, 4], "x": 196, "y": 16 },
+        { "flags": 4, "matrix": [0, 5], "x": 28, "y": 48 },
+        { "flags": 4, "matrix": [0, 6], "x": 84, "y": 48 },
+        { "flags": 4, "matrix": [0, 7], "x": 140, "y": 48 },
+        { "flags": 4, "matrix": [0, 8], "x": 196, "y": 48 }
+        ]
+    },
+    "layouts": {
+        "LAYOUT_2x4": {
+            "layout": [
+                { "label": "enc", "matrix": [0, 0], "w": 1, "x": 3, "y": 0 },
+
+                { "label": "1", "matrix": [0, 1], "w": 1, "x": 0, "y": 1 },
+                { "label": "2", "matrix": [0, 2], "w": 1, "x": 1, "y": 1 },
+                { "label": "3", "matrix": [0, 3], "w": 1, "x": 2, "y": 1 },
+                { "label": "4", "matrix": [0, 4], "w": 1, "x": 3, "y": 1 },
+                { "label": "5", "matrix": [0, 5], "w": 1, "x": 0, "y": 2 },
+                { "label": "6", "matrix": [0, 6], "w": 1, "x": 1, "y": 2 },
+                { "label": "7", "matrix": [0, 7], "w": 1, "x": 2, "y": 2 },
+                { "label": "8", "matrix": [0, 8], "w": 1, "x": 3, "y": 2 }
```

- amend layout name to conform to PR Contributing guidelines
- default key width, for layout, is 1 so thankfully `"w": 1` isn;t required
```suggestion
        "LAYOUT": {
            "layout": [
                { "label": "enc", "matrix": [0, 0], "x": 3, "y": 0 },
                { "label": "1", "matrix": [0, 1], "x": 0, "y": 1 },
                { "label": "2", "matrix": [0, 2], "x": 1, "y": 1 },
                { "label": "3", "matrix": [0, 3], "x": 2, "y": 1 },
                { "label": "4", "matrix": [0, 4], "x": 3, "y": 1 },
                { "label": "5", "matrix": [0, 5], "x": 0, "y": 2 },
                { "label": "6", "matrix": [0, 6], "x": 1, "y": 2 },
                { "label": "7", "matrix": [0, 7], "x": 2, "y": 2 },
                { "label": "8", "matrix": [0, 8], "x": 3, "y": 2 }
```